src/startupscriptgenerator/dotnetcore/main.go

```diff
@@ -19,6 +21,10 @@ func main() {
 		".",
 		"The path to the published output of the application that is going to be run, e.g. '/home/site/wwwroot/'. " +
 		"Default is current directory.")
+	intermediatePathPtr := flag.String(
+		"intermediatePath",
```

Comment:
"intermediatePath" sounds more appropriate to build than run. Maybe runFromPath?